code-challenges.js

```diff
@@ -9,40 +9,88 @@
 
 // Create the code and test each of the variables provided. Expected output: "35 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
-var temp = 35
+//I will create a variable called temp
+//I will use conditionals to determine if the variable "temp" is above, equal to, or below variable boiling point
+
+//Notes on process - I initially had the "else if" statements swapped which would not give me an output of "212 is at boiling point". I swapped them after determining that the code was not properly executing due to the order that the conditionals were in.
+
+// var temp = 35
 // var temp = 350
 // var temp = 212
 
+// var temp = 35
+// if (temp < 212) {
+//   console.log("temp is below boiling point");
+// } else if (temp > 212) {
+//   console.log("temp is above boiling point");
+// } else if (temp = 212) {
+//   console.log("temp is at boiling point");
+// }
+
 
 
 // --------------------2) Create the code that will combine two arrays and return the length using the variables provided below. Expected output: 10
 
-var myNumbers1 = [3, 7, 0, 36, -9]
-var myNumbers2 = [8, -7, 22, 9, 13]
+//I will use the .concat() accessor in order to combine the two arrays
+//I will create a variable that holds the value of both given variables combined
+//I will verify that the arrays combined in terminal output
+//I will create a console log for the new variable comboArrays with a length property attached to it
+//Verified output is 10
 
+//Notes on process - I used the syllabus to read through the concat accessor function as I knew that I needed to combine the two provided variables. Following that, I knew that I needed to find the length of the newly created combined variable, which I also found in the syllabus.
+
+// var myNumbers1 = [3, 7, 0, 36, -9]
+// var myNumbers2 = [8, -7, 22, 9, 13]
+// var comboArrays = myNumbers1.concat(myNumbers2);
+// console.log(comboArrays.length);
 
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variables provided below. Expected output: "eilrahc", "atled"
 
-var myString1 = "charlie"
-var myString2 = "delta"
+//I will combine the two variables with a concat accessor to create a new variable
+//I will use the reverse mutator on the new variable to get the desired output
+
+//Revisit this. Cannot figure out why reverse mutator does not work. I am assuming that reverse only works on arrays, but I was unable to successfully establish the given variables as an array.
```

Comment:
I appreciate this thought process and you are correct. 
There are tools for turning strings into arrays and arrays into strings. 
.split()
.join()

both of these take arguments on how to split and join the data?  in this case:
.split("")
.join("")